lib/puppet/provider/package/dpkg.rb

```diff
@@ -162,4 +164,12 @@ def unhold
     end
   end
 
+  def is_package_installed?(name)
+    unless name.nil? || name.empty?
```

Comment:
Favour `if/else block` over `unless/else block` because is easier to read and understand